terraform/postgresql/provision/main.tf

```diff
@@ -118,7 +118,8 @@ resource "aws_cloudwatch_log_group" "this" {
     create_before_destroy = true
   }
   name              = "/aws/rds/instance/${var.instance_name}/${each.key}"
-  retention_in_days = each.value
+  retention_in_days = each.value["retention_in_days"]
```

Comment:
Also, consider the following variation:

If we don't include the `count` logic in any of the `aws_cloudwatch_log_group` (that is, by always creating the log groups), the logs won't be deleted when disabled (even if `skip_destroy: false`.

The logs will only be destroyed on `cf delete-service`

Reply:
Big suggestion!!! I consider it more appropriate for the client to be aware that they should enable functionality to create log groups instead of being created by default

Reply:
I also added a short explanation about the dependency graph construction and the `depends_on` directive that will help us in the future. Thank you very much @fnaranjo-vmw!!! :rocket: